### PR TITLE
Andrelouis.NokiaInternalTuneEdit version 1.0

### PR DESCRIPTION
**Pull request by upintheairsheep**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/102953)

### CODE REVIEW

**Comment by upintheairsheep** [comment]
portable app

File: manifests/a/Andrelouis/NokiaInternalTuneEdit/1.0/Andrelouis.NokiaInternalTuneEdit.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Andrelouis.NokiaInternalTuneEdit
+PackageVersion: 1.0
+Installers:
+- InstallerUrl: http://onj3.andrelouis.com/phonetones/Software%20and%20Soundbanks/Software/Nokia/Nokia%20Internal%20Tune%20Editor.zip
+  Architecture: x86
+  InstallerType: zip
+  NestedInstallerType: exe
```
Comment (stephengillie) [inline manifests/a/Andrelouis/NokiaInternalTuneEdit/1.0/Andrelouis.NokiaInternalTuneEdit.installer.yaml:10]:
```suggestion
  NestedInstallerType: portable
```

**Comment by stephengillie** [comment]
This package installed normally, but errored when launched:
> Faulting application name: Nite.exe, version: 0.0.0.0, time stamp: 0x39854106
Faulting module name: Nite.exe, version: 0.0.0.0, time stamp: 0x39854106
Exception code: 0xc000041d
Fault offset: 0x00011b90
Faulting process id: 0x0x266C
Faulting application start time: 0x0x1D97EE55D17F494
Faulting application path: C:\Users\User\AppData\Local\Microsoft\WinGet\Packages\Andrelouis.NokiaInternalTuneEdit__DefaultSource\Nite.exe
Faulting module path: C:\Users\User\AppData\Local\Microsoft\WinGet\Packages\Andrelouis.NokiaInternalTuneEdit__DefaultSource\Nite.exe
Report Id: ab9bd053-e06b-47a1-8d42-2447abd4cd21
Faulting package full name: 
Faulting package-relative application ID:

**Comment by upintheairsheep** [comment]
> This package installed normally, but errored when launched:
> 
> > Faulting application name: Nite.exe, version: 0.0.0.0, time stamp: 0x39854106
> > Faulting module name: Nite.exe, version: 0.0.0.0, time stamp: 0x39854106
> > Exception code: 0xc000041d
> > Fault offset: 0x00011b90
> > Faulting process id: 0x0x266C
> > Faulting application start time: 0x0x1D97EE55D17F494
> > Faulting application path: C:\Users\User\AppData\Local\Microsoft\WinGet\Packages\Andrelouis.NokiaInternalTuneEdit__DefaultSource\Nite.exe
> > Faulting module path: C:\Users\User\AppData\Local\Microsoft\WinGet\Packages\Andrelouis.NokiaInternalTuneEdit__DefaultSource\Nite.exe
> > Report Id: ab9bd053-e06b-47a1-8d42-2447abd4cd21
> > Faulting package full name:
> > Faulting package-relative application ID:

Is this Windows Sandbox

**Comment by stephengillie** [comment]
> Is this Windows Sandbox

In a Windows 11 VM.

**Comment by upintheairsheep** [comment]
Try on a normal PC

**Comment by Trenly** [comment]
> Try on a normal PC

This also fails to launch for me on Win10 and Win11. I believe this is because it relies on several files in the same directory, which cannot be called through symlink

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
See https://github.com/microsoft/winget-pkgs/pull/102953#issuecomment-1535545280

I'm seeing this in Hyper-V:
image

**Comment by upintheairsheep** [comment]
Try on windows 10 to see if it works

**Comment by Trenly** [comment]
@upintheairsheep - Support for zipped binaries has now been added and is available in the pipelines. This will require you to update your submission to ManifestVersion: 1.9.0 and set ArchiveBinariesDependOnPath: true in the installer node. To keep things clean, this PR is being closed. Please submit a new PR for the latest version of this package. Thank you for your contributions!

**Pull request closed without merging**